libc-bottom-half/sources/posix.c

```diff
@@ -19,7 +19,7 @@ static int find_relpath2(
 ) {
     // See comments in `preopens.c` for what this trick is doing.
     const char *abs;
-    if (__wasilibc_find_relpath_alloc)
+    if (&__wasilibc_find_relpath_alloc)
```

Comment:
Since we made `__wasilibc_find_relpath_alloc` a strong symbol, it no longer makes sense to do this check, since it'll always be present. The pointer to a function is always non-null.